### PR TITLE
added lists component

### PR DESCRIPTION
**Pull request by saicharan-gouru**
lists

### CODE REVIEW

**Review by bajajcodes — COMMENTED** [review]
- Repeating styles can be clubbed in CSS/lists.css
- Use min-width and max-width both properties, 
   min-width can over-ride max-width property.

File: CSS/lists.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    margin: auto;
+}
+
+.ordered-list {
+    padding: 2rem;
+}
+
+.unordered-list {
+    padding: 2rem;
+}
```
Comment (AshwiniSetty) [inline CSS/lists.css:18]: you can have a class for padding and use those classes wherever required. Else GTG

**Review by AshwiniSetty — APPROVED** [review]

File: Components/Lists/lists.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+                <ol class="ordered-list">
+                    <li>Apple</li>
+                    <li>Ball</li>
+                    <li>Cat</li>
+                </ol>
+
+                <iframe src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%253Col%2520class%253D%2522ordered-list%2522%253E%250A%2520%2520%2520%2520%2520%253Cli%253EApple%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%253Cli%253EBall%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%253Cli%253ECat%253C%252Fli%253E%250A%2520%253C%252Fol%253E"
+                    style="width: 100%; height: 250px; border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin">
+                </iframe>
```
Comment (AshwiniSetty) [inline Components/Lists/lists.html:57]: Instead of using inline styling, you can define a class for displaying code.

**Review by AshwiniSetty — APPROVED** [review]

File: CSS/lists.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital@1&family=Ubuntu:wght@300&display=swap');
+body {
+    font-family: 'Source Sans Pro', sans-serif;
+    font-family: 'Ubuntu', sans-serif;
+}
+
+.lists-container {
+    max-width: 400px;
```
Comment (i-am-a-codegirl) [inline CSS/lists.css:8]: try using rems instead of px..will help you achieve responsiveness faster

File: CSS/lists.css [hunk 4]
```diff
@@ -0,0 +1,29 @@
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital@1&family=Ubuntu:wght@300&display=swap');
```
Comment (i-am-a-codegirl) [inline CSS/lists.css:1]: you can do these imports within you HTML itself..

**Review by i-am-a-codegirl — APPROVED** [review]
Code looks good just gave some suggestions

**Pull request merged**